modules/standard/Map.chpl

```diff
@@ -816,6 +816,7 @@ module Map {
   }
 
   /* Returns a new map containing the keys and values in either a or b. */
+  deprecated "Set-like operators have been deprecated for Map"
```

Comment:
It might be nice to have more individualized messages, in addition to Brad's comment (in the meeting chat).  Especially since operators can be strung together in a single line, it might be difficult for a user to know which one is triggering each message

Reply:
How would you feel about updating them to just say something like `The - operator has been deprecated for map`? That was what I was thinking at first, but found it a bit unsatisfying in the sense that it doesn't offer any alternatives or reasoning, but I also don't want to make the messages too long.

Reply:
That seems reasonable to me!